src/sources/exec/mod.rs

```diff
@@ -57,7 +57,7 @@ pub struct ExecConfig {
     pub streaming: Option<StreamingConfig>,
 
     /// The command to be run, plus any arguments required.
-    #[configurable(metadata(docs::examples = "echo", docs::examples = "Hello World!"))]
+    #[configurable(metadata(docs::examples = r#"["echo", "Hello World!"]"#))]
```

Comment:
Think this is still producing extra quotes?

From the preview:
```
[
  "[\"echo\", \"Hello World!\"]"
]
```

But we'd want probably want something like how [`framing.method`](https://deploy-preview-15502--vector-project.netlify.app/docs/reference/configuration/sources/exec/#framing.method) shows examples?:
```
["echo", "Hello World!"]
```
```
["sleep", "infinity"]
```

Reply:
Yeah, definitely.

I should have left it the way it was before because it was actually displaying correctly but it ended up simply looking weird on the Rust side.

Fixed in d4daa6162.